.ci/Jenkinsfile

```diff
@@ -138,10 +138,10 @@ pipeline {
                 }
             }
             environment {
-                SONARCLOUD_LOGIN = credentials('SONARCLOUD_LOGIN')
+                SONARCLOUD_LOGIN = credentials('\taaf177ed-92f7-4dd7-a482-b6b407be60ae')
             }
             steps {
-                configFileProvider([configFile(fileId: 'xill-platform/settings.xml', variable: 'MAVEN_SETTINGS')]) {
+                configFileProvider([configFile(fileId: '913ac216-d52d-41b0-94f9-cc51e28bd431', variable: 'MAVEN_SETTINGS')]) {
                     sh 'mvn -s "$MAVEN_SETTINGS" -B ' +
                             "-Dsonar.login='${env.SONARCLOUD_LOGIN}' " +
```

Comment:
In `xillio-engine` sonar automatically resolves the host URL and organization through the `SONARCLOUD_LOGIN` variable. I think the same can be done here. Not that it is of importance since it works, but it might be worth looking into.

In Xillio Engine we only have the following parameters:
`-Dsonar.login=${env.SONARCLOUD_LOGIN} -Dsonar.branch.name=${env.BRANCH_NAME}`